python/serialbox/serialization.py

```diff
@@ -292,6 +292,9 @@ def __repr__(self):
     def __str__(self):
         return self.name + '[ ' + ' '.join(['{0}:{1}'.format(*i) for i in self.metainfo.items()]) + ' ]'
 
+    def __eq__(self, other):
+        return self.name == other.name and self.metainfo == other.metainfo
```

Comment:
I'm not entirely convinced that this is correct. 
* On one hand you want to be able to compare whether two Savepoints are the same, like you did above. 
* On the other hand, comparing two Savepoints with each other also means that you want to compare the data associated with the Savepoints. Because fields are always read and written with a Savepoint associated to them.